editor/test/resources/save_data_project/checked.render_target

```diff
@@ -0,0 +1,14 @@
+color_attachments
+{
+    width: 128
+    height: 128
+    format: TEXTURE_FORMAT_LUMINANCE
+}
+
+depth_stencil_attachment
+{
+    width: 128
+    height: 128
+    format: DEPTH_STENCIL_FORMAT_D32F_S8U
+    texture_storage: true
+}
```

Comment:
Don't hand-edit these! They should be saved by the editor.

Reply:
Well, how am I supposed to know that? :)